components/sidebar.js

```diff
@@ -131,25 +119,28 @@ export default ({ hackathons, currentPath }) => {
         <Header>nwPlus CMS</Header>
         {loading && <LoadingImage src={LoadingGif} />}
       </HeaderContainer>
-      <NextLink href="/livesite" as="/livesite" passHref>
+      <NextLink
+        href="/Livesite/announcements"
+        as="/Livesite/announcements"
+        passHref
+      >
         <Link>
           <ItemContainer>
-            <Live color={currentPath === 'livesite' && COLOR.WHITE} />
-            <Label selected={currentPath === 'livesite'}>Livesite</Label>
+            <Live color={currentPath === 'Livesite' && COLOR.WHITE} />
```

Comment:
curious about this capitalization change

Reply:
Looks prettier here LMAO
https://github.com/nwplus/CMS-V2/blob/4e2cc1e103d65a49446520e74f4833c19b5aa6d3/components/page.js#L50